components/Section/TeamSection.tsx

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types'
+import TextSection from '../../components/Section/TextSection'
+import MarkdownTeam from '../../components/Team/MarkdownTeam'
+import DiscourseTeam from '../../components/Team/DiscourseTeam'
+
+function showDiscourseTeam(frontmatter) {
+  return <DiscourseTeam discourseGroupName={frontmatter.discourse_group} />
+}
+
+function showMarkdownTeam(frontmatter) {
+  return <MarkdownTeam members={frontmatter.members} />
+}
+
+const TeamSection = ({ frontmatter }) => {
+  const teamMembers = frontmatter.discourse_group
+    ? showDiscourseTeam(frontmatter)
+    : showMarkdownTeam(frontmatter)
+
+  return (
+    <TextSection
+      classname='pink'
+      title={`Team ${frontmatter.title}`}
+      icon='assistant'
+      iconDirection='center'
+    >
+      <div className='team-members'>{teamMembers}</div>
+      <style jsx>{`
+        :global(.row) {
```

Comment:
I'd avoid global selectors with no scoped paren. This will be completely global and potentilly affects other `.row`s while it's very hard to reason about.